clients/src/main/java/org/apache/kafka/common/network/SslTransportLayer.java

```diff
@@ -168,12 +168,11 @@ public void close() {
             try {
                 socketChannel.socket().close();
                 socketChannel.close();
-            } catch (IOException e) {
-                log.warn("Failed to close SSL socket channel: " + e);
```

Comment:
@ijuma The more annoying exception is the one just above this with the send failure since that occurs very frequently. I haven't seen this warning in logs, so I think keeping this consistent with PlaintextTransportLayer makes sense. I have been meaning to look at graceful termination to avoid the send failure, will raise a JIRA for that.